man/ocitools-generate.1.md

```diff
@@ -45,10 +45,11 @@ read the configuration from `config.json`.
 **--cap-drop**=[]
   Drop Linux capabilities
 
-**--cgroup**=[*PATH*]
-  Use a Cgroup namespace.  If *PATH* is set, join that namespace.  If it
-  is unset, create a new namespace.  The special *PATH* `host` removes
-  any existing Cgroup namespace from the configuration.
+**--cgroup**=*PATH*
+  Use a Cgroup namespace. *PATH* to file of an existing Cgroup namespace.
+  If *PATH* is set, join that namespace. The sepcial *PATH*  ""  set to
+  create a new namespace.  The special *PATH* `host` removes any existing
+  Cgroup namespace from the configuration.
```

Comment:
There's a “sepcial” → “special” typo in here.  And the first two sentences read a bit awkardly to me.  How about:

> Use a Cgroup namespace where *PATH* is an existing Cgroup namespace file to join.  The special empty-string *PATH*  creates a new namespace.  The special *PATH* `host` removes any existing Cgroup namespace from the configuration.